app/spec/factories/aggregator_report.rb

```diff
@@ -0,0 +1,168 @@
+FactoryBot.define do
+  factory :pinwheel_report, class: 'Aggregators::AggregatorReports::PinwheelReport' do
+    transient do
+      pinwheel_service { Aggregators::Sdk::PinwheelService.new(:sandbox) }
+    end
+    identities { [
+      Aggregators::ResponseObjects::Identity.new(
+        account_id: "account1",
+        full_name: "John Smith."
+    ) ]}
```

Comment:
Nit: Idiomatic ruby uses `do`/`end` for multiline blocks. E.g.

```suggestion
    identities do
      [
        Aggregators::ResponseObjects::Identity.new(
          account_id: "account1",
          full_name: "John Smith."
        )
      ]
    end
```